pkgs/applications/video/handbrake/default.nix

```diff
@@ -76,15 +71,26 @@ _EOF
 
     patchShebangs scripts
 
-    substituteInPlace libhb/module.defs \
-      --replace /usr/include/libxml2 ${libxml2.dev}/include/libxml2
-
     # Force using nixpkgs dependencies
     sed -i '/MODULES += contrib/d' make/include/main.defs
     sed -e 's/^[[:space:]]*\(meson\|ninja\|nasm\)[[:space:]]*= ToolProbe.*$//g' \
         -e '/    ## Additional library and tool checks/,/    ## MinGW specific library and tool checks/d' \
         -i make/configure.py
-  '';
+  '' + (if stdenv.isDarwin then ''
```

Comment:
```suggestion
  '' + (lib.optionalString stdenv.isDarwin ''
```

Reply:
Thanks, I thought there was probably a more elegant way to do that.